src/main/java/katsu/tasks/Task.java

```diff
@@ -0,0 +1,82 @@
+package katsu.tasks;
+
+/**
+ * Represents a task with a description and completion status.
+ * This is the base class for different types of tasks in the application.
+ */
+public class Task {
```

Comment:
```suggestion
public abstract class Task {
```
Just an OOP fundamental, since `Task` objects shouldn't be created directly.